frontend/src/metabase/dashboard/actions/core.js

```diff
@@ -54,9 +54,3 @@ export const onReplaceAllDashCardVisualizationSettings = createAction(
   REPLACE_ALL_DASHCARD_VISUALIZATION_SETTINGS,
   (id, settings) => ({ id, settings }),
 );
-
-export const UPDATE_DASHCARD_IDS = "metabase/dashboard/UPDATE_DASHCARD_IDS";
```

Comment:
I added this in the card API endpoint refactor ([PR](https://github.com/metabase/metabase/pull/29762)), but here I'm replacing it with another action `SAVE_CARDS_AND_TABS` (see `tabs.ts`).